druid/src/core.rs

```diff
@@ -212,6 +217,27 @@ impl<T, W: Widget<T>> WidgetPod<T, W> {
         self.state.layout_rect.unwrap_or_default()
     }
 
+    /// Set the viewport offset.
+    ///
+    /// This should be called on a child widget whenever its "physical" position,
+    /// as seen by painting, mouse events, etc. differs from its "logical"
+    /// position. You probably only want this method if you are implementing a
+    /// widget like [`Scroll`].
```

Comment:
I would clarify this further. I think "physical" and "logical" are unclear, here. I would say something like,

`This should must be set by the parent in cases where the parent is modifying the position of a child view relative to the viewport; this is only relevant in special cases such as a scroll view.'

(this doesn't sound much better, either, blah)